### PR TITLE
Slimes can now be pranked.

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
 add: Slimes now have a sense of humor
/:cl:

small meme

### CODE REVIEW

**Review by Crossedfall — APPROVED** [review]
Good changes. Really improves the xenobio experience in ways I never thought was possible.

**Review by AtlasRules — APPROVED** [review]
10/10 - xenobio main

**Comment by spessbro** [comment]
what's ligma

**Comment by AtlasRules** [comment]
joe mama

**Comment by Shadowflame909** [comment]
Make sure to add this to the wiki under slime commands

**Comment by Qustinnus** [comment]
these arent commands

**Comment by Shadowflame909** [comment]
aren't they?

**Comment by Qustinnus** [comment]
its their own fault for falling for it

**Comment by Farquaar** [comment]
This should also include the common variation on the prank that makes reference to the fruit of the ligma tree, known as "ligma nuts"

**Comment by spessbandit** [comment]
but why

**Comment by Farquaar** [comment]
> why

https://www.seriouseats.com/recipes/assets_c/2012/07/20120709-213946-white-bbq-sauce-thumb-625xauto-255585.jpg

**Comment by dr-bee** [comment]
Slimes are now for bulli

File: code/modules/mob/living/simple_animal/slime/life.dm [hunk 1]
```diff
@@ -515,6 +516,16 @@
...
+			else if (findtext(phrase, "ligma balls") && ligma)
+				to_say = "Nooooo...."
+				Target = who
+				ligma = FALSE
+				Friends[who] = -5 //This prank will never be forgotten
+				AIprocess() //Wake up the slime's Target AI, needed otherwise this doesn't work
+			else if (findtext(phrase, "ligma"))
+				to_say = "What's...ligma"
+				ligma = TRUE
+				addtimer(VARSET_CALLBACK(src, ligma, FALSE), 20 SECONDS) //Gotta prank them fast
```
Comment (Fikou) [inline code/modules/mob/living/simple_animal/slime/life.dm]: 20 seconds is too long

**Comment by optimumtact** [comment]
this is cringe

**Comment by PKPenguin321** [comment]
dont ask who Finn is

**Comment by spessbro** [comment]
wow such a good pr getting closed smh orang

**Pull request closed without merging**